src/components/Plugins/ClientConfig/DynamicConfigForm/FormItem.js

```diff
@@ -174,10 +174,10 @@ class DynamicConfigFormItem extends Component {
 }
 
 function mapStateToProps(state, ownProps) {
-  const selectedClient = state.client.selected
+  const selectedClient = state.plugin.selected
```

Comment:
`const selectedPlugin`?

Reply:
(to recap discord convo:) reserving for part 2; many `client` references still within.